style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] Please, make sure you are using the right font family through out your site. It is important that as developers, we follow the given design closely